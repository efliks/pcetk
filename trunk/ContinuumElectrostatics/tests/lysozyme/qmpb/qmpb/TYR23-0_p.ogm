ON_GEOM_CENT 121 2.0
(-7.383 30.652 21.730)  101  1.000
(-7.383 30.652 21.730)  101  0.500
(-7.383 30.652 21.730)  101  0.250
