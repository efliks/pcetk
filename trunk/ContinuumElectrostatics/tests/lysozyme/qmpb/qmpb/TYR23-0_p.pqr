../0/site_TYR_23_p.pqr