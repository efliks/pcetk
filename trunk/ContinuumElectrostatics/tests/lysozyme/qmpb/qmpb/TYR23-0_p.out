Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named TYR23-0_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.919373, 20.3963, 19.0418)) 121 2
Finer level:    (-7.383, 30.652, 21.73) 101 1
Finer level:    (-7.383, 30.652, 21.73) 101 0.5
Finest level:   (-7.383, 30.652, 21.73) 101 0.25


Self energy of TYR23-0_p in three dielectric environment = 138.192
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of TYR23-0_p with background charge set in three dielectric environment = -0.322018
(probably in kcal/mole)



Interaction energies of TYR23-0_p with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.00125203
0   1 : -6.22559e-06
1   0 : -0.000216268
1   1 : 0.00158333
2   0 : -0.00283867
2   1 : -6.86205e-05
3   0 : -0.00683386
3   1 : -0.00182053
3   2 : 0.000536673
3   3 : 0.00459864
4   0 : -0.00302704
4   1 : 0.0131075
5   0 : -0.00109187
5   1 : 0.0123037
6   0 : 276.383
6   1 : 225.157
7   0 : -0.00281969
7   1 : -0.000141688
8   0 : -0.00104448
8   1 : 0.0336005
9   0 : 0.000277118
9   1 : 0.00342722
10   0 : 6.75478e-05
10   1 : 0.0108526
11   0 : -0.000160928
11   1 : 0.00228128
12   0 : -6.26305e-05
12   1 : 0.00152254
13   0 : 1.20434e-05
13   1 : 0.00172051
14   0 : -0.00983254
14   1 : -0.0010761
15   0 : 0.000851716
15   1 : 7.20956e-06
16   0 : -0.000166878
16   1 : 0.00203991
17   0 : -0.0626511
17   1 : 0.00020587
18   0 : -0.000170758
18   1 : 0.00432874
Total runtime: 7.56 s user 0.08 s system 
